setup.py

```diff
@@ -190,6 +190,7 @@ def get_cfbundle_localizations() -> Set:
                 'dataclasses',
                 'shutil',  # Included for plugins
                 'timeout_session',
+                'threading',  # Included for plugins (though it should always be there anyway)
```

Comment:
This too can be removed.  Core code definitely uses it and is highly unlikely to stop that.